### PR TITLE
Added entry for paper accepted as ICAART

### PR DESCRIPTION
**Pull request by ThatGuyIKnow**
Added entry for short paper accepted at ICAART.

@inproceedings{joergensen-et-al-icaart2025,
  author =       "Oliver Joergensen and Vy Le and Tin Nguyen and Ginel Dorleon",
  title =        "Optimizing Elevator Performance with SARL Multi-Agent Systems: A Distributed Approach for Enhanced Responsiveness and Efficiency",
  booktitle =    "Proceedings of the International Conference on Agents and Artificial Intelligence (ICAART)",
  year =         "2025",
  note =         "Short Paper"
}

### CODE REVIEW

File: literatur.bib [hunk 1]
```diff
@@ -9270,6 +9270,14 @@ @InProceedings{jinnai-fukunaga-aaai2017
   crossref =     "aaai2017"
 }
 
+@InProceedings{joergensen-et-al-icaart2025,
+  author =       "Oliver Joergensen and Vy Le and Tin Nguyen and Ginel Dorleon",
+  title =        "Optimizing Elevator Performance with {SARL} Multi-Agent Systems: A Distributed Approach for Enhanced Responsiveness and Efficiency",
+  booktitle =    "Proceedings of the International Conference on Agents and Artificial Intelligence (ICAART)",
+  year =         "2025",
+  note = "Short Paper"
```
Comment (jendrikseipp) [inline literatur.bib]: We usually don't mark short papers, so I suggest to remove the note.

File: literatur.bib [hunk 2]
```diff
@@ -9270,6 +9270,14 @@ @InProceedings{jinnai-fukunaga-aaai2017
   crossref =     "aaai2017"
 }
 
+@InProceedings{joergensen-et-al-icaart2025,
+  author =       "Oliver Joergensen and Vy Le and Tin Nguyen and Ginel Dorleon",
+  title =        "Optimizing Elevator Performance with {SARL} Multi-Agent Systems: A Distributed Approach for Enhanced Responsiveness and Efficiency",
+  booktitle =    "Proceedings of the International Conference on Agents and Artificial Intelligence (ICAART)",
```
Comment (jendrikseipp) [inline literatur.bib]: Can you please add a long entry for the conference in crossref.bib and an abbreviated one in crossref-short.bib? See the other examples as a reference. Also, the booktitle should state "Proceedings of the Xth International ..." with X substituted for the correct number, of course.

File: literatur.bib [hunk 3]
```diff
@@ -9270,6 +9270,14 @@ @InProceedings{jinnai-fukunaga-aaai2017
   crossref =     "aaai2017"
 }
 
+@InProceedings{joergensen-et-al-icaart2025,
+  author =       "Oliver Joergensen and Vy Le and Tin Nguyen and Ginel Dorleon",
+  title =        "Optimizing Elevator Performance with {SARL} Multi-Agent Systems: A Distributed Approach for Enhanced Responsiveness and Efficiency",
```
Comment (jendrikseipp) [inline literatur.bib]:
```suggestion
  title =        "Optimizing Elevator Performance with {SARL} Multi-Agent Systems: {A} Distributed Approach for Enhanced Responsiveness and Efficiency",
```

**Review by jendrikseipp — CHANGES_REQUESTED** [review]
Thanks for the PR. As usual, there are some nitpicks :)

**Comment by ThatGuyIKnow** [comment]
Included requested changes

**Pull request merged**